src/reducers/console.js

```diff
@@ -73,14 +73,19 @@ export default function console(stateIn, {type, payload, meta}) {
       return initialState;
     case 'CONSOLE_INPUT_CHANGED':
       return state.set('currentInputValue', payload.value);
-    case 'CONSOLE_LOG_PRODUCED':
-      return state.setIn(
-        ['history', meta.key],
-        new ConsoleEntry({
-          value: payload.value,
-          evaluatedByCompiledProjectKey: payload.compiledProjectKey,
-        }),
-      );
+    case 'CONSOLE_LOG_BATCH_PRODUCED': {
+      let previousMap = state.get('history');
+      payload.actions.forEach((action) => {
```

Comment:
This isn’t really a collection of _actions_—objects with a `type`, a `payload`, and possibly a `meta`—but rather just a collection of console entry information. So I’d refer to them as something like `entry` heren (and probably destructure as well)

Reply:
Done